distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -255,6 +256,15 @@
                 <include>org.apache.aries.blueprint.core-1.8.2.jar</include>
             </includes>
         </fileSet>
+        <fileSet>
```

Comment:
✏️ Might want to include a comment with a link to the Eclipse ticket where the permanent fix will be tracked.